packages/runtime/src/runtime.ts

```diff
@@ -66,6 +69,38 @@ export class Runtime {
     this.timestamp = 1;
   }
 
+  get defaultBalance (): number {
+    return 1e9; // 1000 Algos
+  }
+
+  /**
+   * Returns a list of initialized default accounts created using static accountSDK from account.ts
+   *  and funded with default balance
+   * @returns list of AccountStore
+   */
+  _setupDefaultAccounts (): AccountStore[] {
+    const balance = this.defaultBalance;
+    const accounts = Object.values(defaultSDKAccounts)
+      .map(accountInfo => new AccountStore(balance, accountInfo));
+    this.initializeAccounts(accounts);
```

Comment:
Do we need to do this here? At line `60` can we intialize all accounts at a single place? Eg. 
```js
this.initializeAccounts([...accounts, ...this._defaultAccounts]);
```

Reply:
I agree, but then every time a `resetDefaultAccount()` method is called the `this.initializeAccounts()` will have to be called inside of it anyways. I don't know which is better

Reply:
The change would look something like that: eg. 
from this:
` resetDefaultAccounts (): void {
    this._defaultAccounts = this._setupDefaultAccounts();
  }`
to this:
` resetDefaultAccounts (): void {
    this._defaultAccounts = this._setupDefaultAccounts();
    this.initializeAccounts(this._defaultAccounts);
  }`

Reply:
ah, ok. Then let's keep what we have currently.